https://bugs.gentoo.org/863671
https://sourceforge.net/p/hdparm/patches/52/

From c8fc52968f9e97d29ccf6deb348ec02627d74235 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 29 Jul 2022 13:00:40 -0700
Subject: [PATCH] hdparm: Remove unused linux/fs.h header

this helps fixing builds with glibc 2.36 where mount.h conflits now with
kernel mount.h which is included indirectly by linux/fs.h see [1] for
details

[1] https://sourceware.org/glibc/wiki/Release/2.36

Upstream-Status: Submitted []
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/hdparm.c
+++ b/hdparm.c
@@ -25,7 +25,6 @@
 #include <sys/mman.h>
 #include <sys/user.h>
 #include <linux/types.h>
-#include <linux/fs.h>
 #include <linux/major.h>
 #include <endian.h>
 #include <asm/byteorder.h>

